### PR TITLE
More translations in Spanish, and minor corrections.

### PR DESCRIPTION
**Pull request by SidVal**
More translations in Spanish, and minor corrections.

### CODE REVIEW

File: translations/es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -491,8 +501,10 @@ msgstr "Unete Ahora"
 #: /content/learn/who-uses-amp.yaml
 msgid ""
 "Join an ecosystem of  850,000 domains, leading platforms, and 100+ technology"
-" providers using  AMP."
+" providers using AMP."
 msgstr ""
+"Uneta al ecosistema de 850 mil dominios, plataformas líderes y más de "
```
Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: Should "Uneta" be "Unete"?

File: translations/es/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -866,11 +880,11 @@ msgstr "Plataformas, Proveedores y Socios Soportados"
 
 #: /content/includes/who.yaml
 msgid "Technology Companies using AMP"
-msgstr ""
+msgstr "Compañías Tecnológicas usando AMP"
 
 #: /content/docs/build.md
 msgid "Templates"
-msgstr ""
+msgstr "Diseños"
```
Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: Should this be "Plantillas"?

File: translations/es/LC_MESSAGES/messages.po (hunk 1 above)
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Yes, now it's fixed. Isn't it?

File: translations/es/LC_MESSAGES/messages.po (hunk 2 above)
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: All right. Fixed

**Pull request merged**